### PR TITLE
Personal portfolio project for Microverse

### PR DESCRIPTION
**Pull request by Jusha25**
Personal porfolio project
#Built in: Html, css, Linters.

This project can be cloned and run on local machine. It is mobile website version and made on second (secondpage) branch of personal portfolio project and made by using Flexbox and Grid Layout.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,47 +1,211 @@
...
+              <img class="piling" src="./images/htmllogo.png" alt="" />Html
+            </li>
+            <li><img class="piling" src="./images/csslogo.png" alt="" />Css</li>
+          </ul>
+        </li>
+        <li><h2>Frameworks</h2></li>
+        <li><h2>Skills</h2></li>
+      </ul>
+      <hr />
+    </section>
```
Comment (cwaku) [inline index.html:192]: - [ ] As I stated with the other requirement it's very important that is at its best. Let's follow the design template to make our work look awesome. 👍🏾 

<html>
<body>
<p dir="auto" style="box-sizing: border-box; margin-top: 0px !important; margin-bottom: 16px; color: rgb(201, 209, 217); font-family: -apple-system, BlinkMacSystemFont, &quot;Segoe UI&quot;, Helvetica, Arial, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;; font-size: 14px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(13, 17, 23); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;">Please ignore the green and orange borders around the elements</p>

Figma design | Your design
-- | --
image |  image



</body>
</html>

**Review by cwaku — CHANGES_REQUESTED** [review]
Hi @Jusha25, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there! Implement these changes and you'll be in a good spot to get your project approved. 💪🏾 

## Highlights
✅ Correct use of Github Flow 👍🏾 
✅ No linter errors 💯 
✅ Descriptive Pull Request 🥇 



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -1,47 +1,270 @@
...
+      </ul>
     </section>
-</html>
\ No newline at end of file
+
+    <!-- ------Middle------section---- -->
+    <section class="works">
+      <section>
+        <img src="./images/snapshoot-portfolio.png" alt="" />
+        <h1 class="section-title">Tonic</h1>
```
Comment (mikethreels) [inline index.html:83]: - [ ] there can only by 1 'h1' tag on your page.

File: index.html [hunk 3]
```diff
@@ -1,47 +1,270 @@
...
+        <div class="section-about">
+          <a href="#" class="canopy">CANOPY</a>
+          <img src="./images/Counter.png" alt="" class="size" />
+          <h4 class="size back-end">Back End Dev</h4>
+          <img src="./images/Counter.png" alt="" class="size dot" />
+          <h4 class="year">2015</h4>
+        </div>
+        <p>
```
Comment (mikethreels) [inline index.html:91]: - [ ] the content of each card in the works class should be aligned with flexbox.

File: index.html [hunk 4]
```diff
@@ -1,47 +1,270 @@
...
+          <a
+            href="https://www.linkedin.com/feed/"
+            class="twitter-icon icon-right"
+          >
+            <img src="./images/twitter.png" alt="" />
+          </a>
+        </li>
+        <li>
+          <a
+            href="https://www.linkedin.com/feed/"
```
Comment (mikethreels) [inline index.html:193]: - [ ] the content of the about myself section should be aligned using flexbox

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Shaili, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by elmar8287 — APPROVED** [review]
Hi, 

# APPROVED  👍 🥇 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Jusha25** [comment]
thank you very much

**Review by Qoosim — COMMENTED** [review]
# **Status: Approved** :tada: 

Hi @Jusha25 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- The Pull Request is following the GitHub Flow :heavy_check_mark: 
- The PR title and description are professional and descriptive :heavy_check_mark: 
- The commit messages are meaningful :heavy_check_mark:  
- Linters are working properly :heavy_check_mark: 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**